server/emails/emailTemplate.js

```diff
@@ -0,0 +1,43 @@
+import Mailgen from "mailgen";
+
+const mailGenerator = new Mailgen({
+  theme: "default",
+  product: {
+    name: "Authors Haven",
+    link: "#"
+  }
+});
+
+// Prepare email contents
+/**
+ * @param  {String} firstName
+ * @param  {String} lastName
+ * @param  {String} url
+ * @methods passwordResetTemplate
+ */
+
+const emailTemplate = (
```

Comment:
If this is a method name it like a method e.g.` generateEmailTemplate` or `getEmailTemplate`